Automattic/wp-calypso
# Shortcode: Type Gallery shortcode

- Remove `propTypes` access from production runtime.
- Move to TypeScript
- No functional changes

Part of #35695

## Testing

- Use the Calypso classic editor.
- Add a Gallery (add media and select multiple images).
- Does the Gallery work as expected?
